.gitignore

```diff
@@ -43,3 +43,6 @@ jspm_packages
 .env*
 
 .cache
+
+
+.DS_Store
```

Comment:
I would remove this for these reasons

https://jaketrent.com/post/gitignore-system-files/